### ISSUE TITLE
How to configure to return to camera view after accepting photo in previewer

### ISSUE THREAD

**Issue by splons** [Mijick/CameraView#34]
I have an application where the user will likely want to take multiple photos in a session. I would like to configure so that when they accept a photo in the preview it takes them back to the live photo screen to take another. Thank you.

**Comment by FulcrumOne** [comment]
Hey @splons,

Ahh, apologies, seems I forgot to implement this obvious use case. 
I'm adding this request to the [backlog](https://mijick.notion.site/ed227d35caa249e4a04e3a1e93129576?v=013c9f8962744b9f84bf4b3a544196db&pvs=74).

Best regards,
Tomasz

**Comment by FulcrumOne** [comment]
hey @splons,

The feature was implemented in the`patch-2.1.0` branch. Could you check whether it works as expected?

Here's an example of how it works:
```Swift
MCameraController(manager: manager)
    .afterMediaCaptured { $0
        .returnToCameraView(true)
        .closeCameraController(false)
        .custom { print("Something") }
    }
```

**Comment by splons** [comment]
Hello, thank you for the timely work on this. I am traveling until Tuesday. I look forward to trying it asap I get back home. SteveOn Jul 27, 2024, at 12:22, Tomasz K. ***@***.***> wrote:﻿
hey @splons,
The feature was implemented in thepatch-2.1.0 branch. Could you check whether it works as expected?
Here's an example of how it works:
MCameraController(manager: manager)
    .afterMediaCaptured { $0
        .returnToCameraView(true)
        .closeCameraController(false)
        .custom { print("Something") }
    }

**Comment by splons** [comment]
@FulcrumOne thank you for the fast action on this request. Tried it out and it works exactly as expected. Much appreciated!
